### PR TITLE
Update Shapely

### PR DESCRIPTION
**Pull request by alimcmaster1**
Ref: https://github.com/neo4j-contrib/neomodel/pull/607

cc. @[joekohlsdorf](https://github.com/joekohlsdorf)

From the code looks it seems like Shapely is really an optional dependency of neomodel, since alot of functionality can be used without it. Similar to optional dependencies in pandas. https://pandas.pydata.org/docs/getting_started/install.html
Ref: https://github.com/neo4j-contrib/neomodel/blob/master/neomodel/contrib/spatial_properties.py#L26

@whatSocks @nossila to review

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,9 +18,10 @@
              'scripts/neomodel_remove_labels'],
     setup_requires=[
         'pytest-runner'] if any(x in ('pytest', 'test') for x in sys.argv) else [],
-    tests_require=['pytest>=6.2.4', 'shapely', 'neobolt'],
```
Comment (joekohlsdorf) [inline setup.py:21]: Is it ok to remove `neobolt` from the test requirements?

File: setup.py [hunk 2]
```diff
@@ -18,9 +18,10 @@
...
+    tests_require=['pytest>=6.2.4'],
```
Comment (joekohlsdorf) [inline setup.py:21]: Is it ok to remove `neobolt` from the test requirements?

Comment (alimcmaster1) [reply setup.py:21]: neobolt is an install requirement - so will be installed from there :) test requirements should generally be additional requirements just needed for testing e.g pytest etc..

**Comment by alimcmaster1** [comment]
Friendly ping to maintainers here - @whatSocks @nossila to review

**Comment by alimcmaster1** [comment]
From speaking to maintainers it would seem this change is correct. Shapely is really an optional dependency and not a hard requirement as per the comment: https://github.com/neo4j-contrib/neomodel/blob/master/neomodel/contrib/spatial_properties.py#L26

**Pull request merged**